frontend/src/scenes/billing/BillingLocked.tsx

```diff
@@ -5,7 +5,7 @@ import { BillingSubscribedTheme } from './BillingSubscribed'
 import { compactNumber } from 'lib/utils'
 import { LemonButton } from '@posthog/lemon-ui'
 import { IconCancel } from 'lib/components/icons'
-import { BillingLockedV2 } from './v2/BillingLocked'
+import { BillingLockedV2 } from './v2/control/BillingLocked'
```

Comment:
I thought we were redoing just `Billing.tsx`, you can keep this file out of the experiment as it just wraps `Billing.tsx`

Reply:
Worth keeping in the experiment only if you think we're going to change the Locked screen, which for now basically means forcing the user to be on the billing page if they need to be locked out. So maybe I'd keep it out of the experiment for now.